`ATMMetaForce` objects are not serializeable

The `ATMMetaForce` object does not register a serialization proxy, and so instances of it and any systems containing such a force are not serializable using the OpenMM `openmm.XmlSerializer.serialize` interface.

Trying to serialize just the force, for example,

```
import atmmetaforce
import openmm

force = atmmetaforce.ATMMetaForce(0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1, [])
openmm.XmlSerializer.serialize(force)
```

will produce the following error:

```
object = <atmmetaforce.ATMMetaForce; proxy of <Swig Object of type 'ATMMetaForcePlugin::ATMMetaForce *' at 0x16fd27360> >

    @staticmethod
    def _serializeForce(object):
        r"""_serializeForce(object) -> std::string"""
>       return _openmm.XmlSerializer__serializeForce(object)
E       openmm.OpenMMException: There is no serialization proxy registered for type N18ATMMetaForcePlugin12ATMMetaForceE
```

--- a/serialization/src/ATMMetaForceProxy.cpp
+++ b/serialization/src/ATMMetaForceProxy.cpp
@@ -25,7 +25,7 @@ void ATMMetaForceProxy::serialize(const void* object, OpenMM::SerializationNode&
     node.setDoubleProperty("aCore", force.getDefaultAcore());
     node.setDoubleProperty("direction", force.getDefaultDirection());
 
-    OpenMM::SerializationNode& variableForceGroups = node.createChildNode("GlobalParameters");
+    OpenMM::SerializationNode& variableForceGroups = node.createChildNode("VariableForceGroups");
     for (const auto i : force.getVariableForceGroups()) {
         variableForceGroups.createChildNode("Parameter").setIntProperty("group", i);
     }
@@ -47,7 +47,7 @@ void* ATMMetaForceProxy::deserialize(const OpenMM::SerializationNode& node) cons
     try {
         std::vector<int> variableForceGroups;
 
-        const OpenMM::SerializationNode& globalParams = node.getChildNode("GlobalParameters");
+        const OpenMM::SerializationNode& globalParams = node.getChildNode("VariableForceGroups");
         for (auto& parameter : globalParams.getChildren())
             variableForceGroups.push_back(parameter.getIntProperty("group"));
 

